src/vtbackend/Screen_test.cpp

```diff
@@ -1580,6 +1580,11 @@ TEST_CASE("ScrollUp.WithMargins", "[screen]")
                 "KLMNO\n"
                 == screen.renderMainPageText());
     }
+    mock.writeToScreen("\033[r");
+    mock.writeToScreen("\033[s");
+    REQUIRE(screen.margin().vertical == Margin::Vertical { .from = LineOffset(0), .to = LineOffset(4) });
+    REQUIRE(screen.margin().horizontal
+            == Margin::Horizontal { .from = ColumnOffset(0), .to = ColumnOffset(4) });
```

Comment:
This is C++20 syntax. We do also compile with C++20 here and there on CI, but we still need to support C++17, at least selectively.

You could make this test depending on the `__cplusplus` version. Not sure that's the best solution.

On the other hand, we can evaluate if (and if not => when) it makes sense to bump the minimum C++ version to C++20.

Reply:
I'll use the regular syntax here. I just didn't know we should be compatible with c++17.